content/raw/wallet/2.0.0_send_transaction.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Send Transaction spec"
+description: ""
+version: "2.0.0"
+date: 2022-05-10T12:03:57+00:00
+lastmod: 2022-05-10T12:03:57+00:00
+draft: false
+images: []
+menu:
+  raw:
+    parent: "wallet"
+weight: 100
+toc: true
+---
+
+# Send Transaction Spec
+
+## Motivation
+
+One of the most common use cases for a wallet is to send tokens to another account. A user typically expects to know how much tokens they are sending are worth in some currency and also expect a wide range of functionality such as ENS addresses, address validation using EIP-55, and the ability to choose easily common addresses, saved or recently used address to send to. The user also expects to be able to understand the transactions fees involved and choose the priority for transaction.
+
+## Definitions
+
+### Best route
+
+We define the best route as the less costly route amongst all the possible route
```

Comment:
wouldn't this be a time to less costly trade off?

Reply:
I was thinking as an option here to either choose between time or cost

Reply:
optimal I thought would be best tradeoff between time and cost. like perhaps not the cheapest but something with better time than the cheapest option? 

I don't see design for the user to make choice between cost and time yet, right?

Reply:
I thought about that too, choosing something having the best trade off, this means we have to bias our algorithm. How do u compare time and cost? is 10 gwei  for 5 minutes better than 20 gwei for 2 minutes 30 seconds?

As u can see we would have to apply a multiplicator between time and cost. TBH I have no idea what it should be I even believe it is different depending on who u are. So I kinda avoid to make a decision by having this option

WDYT?

> I don't see design for the user to make choice between cost and time yet, right?

No design yet indeed, I was thinking it is only a toggle into wallet settings

Reply:
Resolved: best route in terms of cost in a reasonable time